index.js

```diff
@@ -22,7 +22,7 @@ function stubMethod(service, method, replacement) {
   if (!isStubbed(service)) stubService(service);
   if (!replacement) return sinon.stub(getService(service).prototype, method);
 
-  return sinon.stub(getService(service).prototype, method, function(params, callback) {
+  return sinon.stub(getService(service).prototype, method).callsFake(function(params, callback) {
```

Comment:
Without this adjustment and with the sinon upgrade, does this break? That is, would upgrading here break existing tests in other repos that provide a fake function in this same way?